microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -19,6 +19,7 @@ The web is constantly evolving to improve the user experience, security, and pri
 | Referrer Policy: Default to `strict-origin-when-cross-origin` | Canary/Dev experimentation starting in M79 | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus6251880185331712].  Based on user and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge at the same time or later than Google.  |  
 | Disallow synchronous XmlHttpRequest in page dismissal | Shipping in M82 | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus4664843055398912].  Based on enterprise and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge later than Chrome.  Matching Chrome, Edge offers a Group Policy to disable this change until Edge 88.  |  
 | Disable TLS/1.0 and TLS/1.1 by default | Shipping in M82 | The `edge://flags/#show-legacy-tls-warnings` flag may be set to cause Edge to show a non-blocking "Not Secure" notice when loading pages that require legacy TLS protocols.  The [SSLMinVersion][DeployedEdgePoliciesSSLMinVersion] Group Policy permits re-enabling of TLS/1.0 and TLS/1.1; the policy remains available until Edge 88.  | 
+| Show a subtle prompt for notification permission requests | Canary experimentation starting in M83 | When sites request to send users notifications via the `Notifications` or `Push` APIs, Edge shows a subtle request icon in the address bar, instead of the flyout permission prompt. This experiment applies to all users on all sites which request notifications permissions, unless the user opts out via `edge://settings/content/notifications`. In the future, we may explore showing the full flyout prompt in certain situations based on user behaviors and other signals. | 

```

Comment:
```suggestion
| Display subtle prompt for notification permissions requests | Available in Canary branch v83 or later | Microsoft Edge displays a subtle request icon in the address bar instead of the permission flyout prompt.  The subtle request icon is displayed when a site requests to send future notifications to the user using the `Notifications` or `Push` API.  This experiment applies to all sites that request notifications permissions from any user.  The user may opt out using `edge://settings/content/notifications`.  In a future release, the full flyout prompt may be implemented for specific situations based upon user behavior and other input. |  

```